lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -8318,10 +8345,7 @@ bool SwiftASTContextForExpressions::CacheUserImports(
 
   auto src_file_imports = source_file.getImports();
 
-  Progress progress(llvm::formatv("Caching Swift user imports from '{0}'",
-                                  source_file.getFilename().data()),
-                    src_file_imports.size());
-
+  Progress progress("Caching Swift user imports from");
```

Comment:
This is very LLDB-centric language:
"Importing module used in expression"

Reply:
I can change this, but do we still want to tell users that Swift imports were cached?

Reply:
No, the existing wording confused even me (as someone familiar with the implementation) and I had to look at the code to understand what "caching" in this context is even supposed to mean.

Reply:
I don't think we can achieve that today since progress report can't be related one another. You can stack them but that don't necessarily means the one at the bottom of the stack is a parent of the one above. It would be a nice improvement for later though :)

Reply:
Yeah it would be nice to have this in the future 👍🏾